db/schema.rb

```diff
@@ -638,6 +638,7 @@
     t.boolean  "exporting_photos",                                 default: false
     t.string   "color_theme",                        limit: 255
     t.boolean  "post_default_public",                              default: false
+    t.boolean  "chat_auto_login",                                  default: false
```

Comment:
Well we could create a configuration option. Make it false on default and if the podmin likes to do it differently he can adjust the diaspora.yml, no?